velox/type/DecimalUtil.h

```diff
@@ -32,5 +32,34 @@ class DecimalUtil {
   /// Helper function to convert a decimal value to string.
   template <typename T>
   static std::string toString(const T& value, const TypePtr& type);
+
+  inline static int128_t rescaleWithRoundUp(
+      const int128_t& unscaledValue,
```

Comment:
passing by value should be better.

Reply:
But, we are not modifying this value. We need one copy to find the remainder.

Reply:
I meant there is no huge benefit to passing an `in128_t` value by reference. Passing by value is simpler from an API standpoint.